examples/projections/nongeo/cartesian_logarithmic.py

```diff
@@ -0,0 +1,37 @@
+"""
+Cartesian logarithmic
+=====================
+
+``xwidth[l]/[height[l]]``: Give the ``width`` of the figure and the optional argument \
+``height``. The axis or axes with a logarithmic transformation requires ``l`` after
+its size argument.
+"""
+import pygmt
+import numpy as np
+
+# Create a list of x values 0-100
+xline = np.arange(0, 101)
+# Create a list of y-values that are the square root of the x-values
+yline = xline ** 0.5
+# Create a list of x values for every 10 in 0-100
+xpoints = np.arange(0, 101, 10)
+# Create a list of y-values that are the square root of the x-values
+ypoints = xpoints ** 0.5
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[1, 100, 0, 10],
+    # Set a logarithmic transformation on the x-axis
+    projection="x4cl/0.4c",
```

Comment:
Any reason for lower-case `x` (map scale), not upper-case `X` (map width)?

Reply:
The example for the Cartesian power projection in the GMT docs used the lower case x, so I copied it from there. I'll try it with the capital x and change it to be consistent with the rest of the projection examples.

Reply:
I pushed a change using map width